src/test/java/com/google/cloud/tools/maven/it/RunMojoIntegrationTest.java

```diff
@@ -86,6 +97,34 @@ public void run() {
     assertEquals("Hello from the App Engine Standard project.", urlContent.toString());
     verifier.verifyErrorFreeLog();
     verifier.verifyTextInLog("Dev App Server is now running");
+    verifier.verifyTextInLog("Module instance " + expectedModuleName + " is running");
+  }
+
+  @SuppressWarnings("unused") // used for testRun()
```

Comment:
Ah okay, can you add a comment just linking it to the test method? like`@{link whatever}`?